src/js/components/help/daimsMessage.jsx

```diff
@@ -12,18 +12,18 @@ export default class DaimsMessage extends React.Component {
 
     render() {
         let type = <span>.</span>;
-        let version = 'v1.1';
+        let version = 'v2.0';
         if(this.props.type == 'practices') {
             type = <span> and its <a href='https://community.max.gov/download/attachments/1286474850/DAIMS_Practices_Procedures_v1.1.pdf?version=1&modificationDate=1498857477698&api=v2' className='daims-link'>Practices &amp; Procedures {version}.</a></span>
         }
 
         return (
             <div className='daims'>
-                <span className='daims-header'>June 30, 2017: </span>
+                <span className='daims-header'>September 27, 2017: </span>
                 <span className='daims-message'>
                     Treasury released the DATA Act Information Model Schema (DAIMS) {version}.
-                    DAIMS {version} is a minor update of the schema and will be implemented in production in the fall of 2017.
-                    Find out more information about <a className='daims-link' href='https://community.max.gov/x/YgyuT'>DAIMS {version}</a>{type}
+                    DAIMS {version} is a major update of the schema and will be finalized in December 2017 for implementation in Q3 FY 2018.
```

Comment:
Finalized in Dec 2017 for use in FY of 2018? That doesn't sound right, are we sure that's right?

Reply:
Thats the language in the docs I was provided. Will attach to Jira ticket